### DISCUSSION TITLE
MY ACCOUNT HAS BEEN HACKED

### DISCUSSION THREAD

**Discussion by paaavannn7** [community/community#128120] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

General

### Body

This is my second account I needed this to get support. My primary account is hacked, someone logged in my account and changed password and email. Please help me to get back my account. I can provide you any information you need.

**Comment by SjxSubham** [comment]
Change the password as soon as possible and activate 2 factor authorization

**Reply by paaavannn7** [reply]
As hacker has changed my email and password, I do not have access to it.

**Comment by Akshitha-reddy-13** [comment, marked as answer]
Better still, contact GitHub help (https://support.github.com/request). In such cases, the community cannot provide meaningful assistance; however, they will guide you in the best possible manner.

**Reply by paaavannn7** [reply]
I will try that. Thanks

**Reply by Akshitha-reddy-13** [reply]
I hope that helps you to resolve the issue.

**Comment by ebndev** [comment]
Hi @paaavannn7,

As mentioned by another user, this community is managed by GitHub employees, but it is not a support forum. Please open a ticket on our [Support page](https://support.github.com/contact) if you haven't already done so.

This is a public forum, and the community is unable to assist with nor escalate your ticket, but we can assure you it is in the right place. It will be answered in the order it was received based on our team's resources and the amount of tickets ahead of yours.

As no other Community Discussions team members will be able to provide additional help with account related questions I am going to close this discussion. We appreciate your understanding!
